scripts/e2e-tests.sh

```diff
@@ -11,4 +11,4 @@ cargo +"$NIGHTLY_TOOLCHAIN" build --release --target wasm32-unknown-unknown -Z b
 export RPC_URL=http://localhost:8547
 # We should use stable here once nitro-testnode is updated and the contracts fit
 # the size limit. Work tracked [here](https://github.com/OpenZeppelin/rust-contracts-stylus/issues/87)
-cargo +"$NIGHTLY_TOOLCHAIN" test --features std,e2e --test "*"
+cargo +"$NIGHTLY_TOOLCHAIN" test --features e2e -p safe-erc20-example --test safe-erc20
```

Comment:
Will revert this once ready for merge, using this for easier testing

Reply:
Yeah. If yours IDE allows to set build settings for tests, you can also run specific integration test from IDE's UI.

Reply:
Regarding the [msg sender issue above](https://github.com/OpenZeppelin/rust-contracts-stylus/pull/289#issuecomment-2362200392), I hope the explanation was clear enough. The same issue is there for `safe_transfer_from`, as the msg sender isn't the one expected (the original sender of tx).